src/SignalR/test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/JsonHubProtocolTests.cs

```diff
@@ -119,7 +119,7 @@ public void WriteMessage(string protocolTestDataName)
 
             var expectedOutput = Frame(testData.Json);
 
-            var protocolOptions = new JsonHubProtocolOptions
+            var protocolOptions = new NewtonsoftJsonHubProtocolOptions
```

Comment:
Rename JsonHubProtocolTests.cs -> NewtonsoftJsonHubProtocolTests.cs

Reply:
I was going to use this file for both NewtonsoftJson and System.Text.Json when we add that.